packages/familie-header/package.json

```diff
@@ -30,31 +21,40 @@
   },
   "dependencies": {
     "@navikt/boxed-list-with-links": "^1.0.2",
-    "@navikt/ds-icons": "^0.1.5",
+    "@navikt/ds-icons": "^0.1.6",
     "@navikt/familie-form-elements": "^2.2.0",
     "@navikt/familie-ikoner": "^3.1.9",
     "@navikt/familie-typer": "^3.0.12",
-    "@navikt/nap-user-panel": "^1.0.0",
+    "@navikt/nap-user-panel": "^1.1.0",
     "classnames": "^2.2.6",
     "lodash.throttle": "^4.1.1",
-    "nav-frontend-alertstriper": "^3.0.13",
-    "nav-frontend-alertstriper-style": "2.0.33",
-    "nav-frontend-core": "^5.0.8",
-    "nav-frontend-ikoner-assets": "2.0.8",
-    "nav-frontend-js-utils": "1.0.15",
-    "nav-frontend-knapper": "^2.0.31",
-    "nav-frontend-knapper-style": "^1.0.16",
-    "nav-frontend-paneler": "^1.0.24",
-    "nav-frontend-paneler-style": "0.3.25",
-    "nav-frontend-popover": "0.0.40",
-    "nav-frontend-popover-style": "0.0.8",
-    "nav-frontend-skjema": "^3.0.52",
-    "nav-frontend-typografi": "^2.0.17",
-    "nav-frontend-typografi-style": "^1.0.18",
+    "nav-frontend-alertstriper": "^3.0.51",
+    "nav-frontend-alertstriper-style": "2.0.38",
+    "nav-frontend-core": "^5.0.11",
+    "nav-frontend-ikoner-assets": "2.0.10",
```

Comment:
Er det meningen at disse nav-dependenciene skal være låst? Altså burde det ikke være `^2.0.10` for eksempel?

Reply:
Jo, det kan du si :) Er potensiale for å forbedre dette, har ikke sett så nøye på det 💯